virtcontainers/container.go

```diff
@@ -435,6 +436,16 @@ func (c *Container) mountSharedDirMounts(hostSharedDir, guestSharedDir string) (
 			return nil, err
 		}
 
+		for _, devInfo := range c.config.DeviceInfos {
+			if devInfo.DevType == "e" && devInfo.ContainerPath == m.Source {
+				// We are not going to bind mount host ephemeral directory
+				// provided by k8s, instead at later stage we are going to
+				// create a tmpfs inside a VM and use it to bind mount to the
+				// containers of the pod.
+				continue MntLoop
```

Comment:
In order to avoid the `goto` here, please use something like this:
```go
                loopOver := false
                for _, devInfo := range c.config.DeviceInfos {
			if devInfo.DevType == "e" && devInfo.ContainerPath == m.Source {
				// We are not going to bind mount host ephemeral directory
				// provided by k8s, instead at later stage we are going to
				// create a tmpfs inside a VM and use it to bind mount to the
				// containers of the pod.
				loopOver = true
                                break
			}
		}

                if loopOver {
                            continue
                }
```